src/views/unauthenticated/About.tsx

```diff
@@ -0,0 +1,159 @@
+						</svg>
+					</Link>
+				</div>
+			</section>
+			<section>
+				<h2>Thank you</h2>
+				<p>Mentors: Alex, Aditya, Tanner</p>
+				<p>The entire The Collab Lab.</p>
```

Comment:
<3 

Extra mile would be to link to everyone's relevant profiles (github, maybe?) Most of that can be found here: https://github.com/the-collab-lab/tcl-77-smart-shopping-list/blob/main/CONTRIBUTORS.md

Reply:
Yes please! I am happy to provide my GitHub profile link and my LinkedIn link again!
https://www.linkedin.com/in/ross-clettenberg/
https://github.com/RossaMania

Reply:
I'm glad everyone likes it! 😊 I didn't think about adding the mentors links 🤦🏽‍♀️, I did think about the logos as the links for the collabies so a nice little update is needed!

Reply:
this whole unauthenticated feature is such a great idea overall!!! It really adds a lot of value for the users and creates a layer to the application that just makes so much sense for a professional application!!! amazing job!!!